ui/src/components/MainContainer.js

```diff
@@ -127,6 +127,8 @@ const MainContainer = () => {
             't3ch',
             'admin',
             'stewardship',
+            'tc',
```

Comment:
What's the diff between this and #114

Reply:
hm i dont get it? this is just to allow TC and MT users to still allow them use their profiles as normal. because I tested with someone in TC and they can access `/admin/forms` but not `/profile`

Reply:
My bad, didn't actually comment on the offending line. Both #114 and #122 have the same link but different permissions which doesn't make sense to me

Reply:
oooo ok hmm. cmiiw but is this because some forms requires login and some are not? @ksuhartono97 
![image](https://github.com/hmcc-global/hmcchk-web/assets/26876443/7796b919-0185-4f78-b258-738daf654056)